app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Party.scala

```diff
@@ -19,7 +19,7 @@ package uk.gov.hmrc.cdsimportsddsfrontend.domain
 case class Party(
                   name: Option[String] = Some("French Foil Ltd."),
                   identifier: Option[String] = Some("GB225456994000"),
-                  address: Option[Address] = Some(Address()),
+                  address: Option[Address] = None,
```

Comment:
default value needed for address? I think, we were planning to remove default values from domain models.

Reply:
we've moved some defaults from the domain models into the viewmodels, but haven't completely cleaned up Party due to the size of the task